atd-vze/src/views/Crashes/Recommendations/recommendationsDataMap.js

```diff
@@ -20,10 +20,5 @@ export const recommendationsDataMap = {
       label: "Updates",
       key: "rec_update",
     },
-    partner_id: {
-      label: "Coordination Partner:",
-      lookupOptions: "atd__coordination_partners_lkp",
-      key: "coord_partner_desc",
```

Comment:
this was a duplication pretty much of the first object and its not actually doing anything?